### ISSUE TITLE
Issue using private packagist

### ISSUE THREAD

**Issue by Morgy93** [yireo/magento2-replace-tools#13]
Hello,

upon use of `composer replace:bulk:add yireo/magento2-replace-inventory` I get the following error:
```
In AuthHelper.php line 205:
                                                                                                                             
  The 'https://repo.packagist.com/foo/p2/yireo/magento2-replace-inventory~dev.json' URL required authentication (HTTP 401).  
  You must be using the interactive console to authenticate
```

I never had issues with the authentication to that repo. The credentials are available in the `./auth.json`.
Plus, I don't know how to use the interactive console. 🙈 

Any help?

**Comment by jissereitsma** [comment]
The reason this issue arises is because the composer replace tools are trying to find the package `yireo/magento2-replace-inventory` in any of the configured repositories, in your case your Packagist.com account. But because the package is **not** found in Packagist.com this error is thrown. Most likely this is due to the ordering of your repositories. But the error should not occur.

Could you upgrade to 1.0.6 to see if this fixes things?

**Comment by Morgy93** [comment]
Yes, fixed with 1.0.6, thanks!
